### PR TITLE
Add command-line args

### LINKED ISSUE #36: Main should have a framework for accepting command-line args and passing them as game configs

### PR DESCRIPTION
**Pull request by nathanshelly**
Only current accepted argument is `food-count` to specify the number of
pieces of food on-screen at once.

Closes #36

[asciicast](https://asciinema.org/a/JkRlWJ4EzElveKDeTFadtB6F5)

### CODE REVIEW

**Comment by nathanshelly** [comment]
Will merge this after #38 is merged since there will be some hairy conflicts there

File: Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 [package]
 name = "snek"
 version = "0.1.0"
-authors = ["sashaweiss <asashaweiss@gmail.com>"]
+authors = ["sashaweiss <asashaweiss@gmail.com>", "nathanshelly <nsnshelly@gmail.com>"]
```
Comment (sashaweiss) [inline Cargo.toml:4]: get outta here

File: src/snek/driver.rs [hunk 2]
```diff
@@ -163,3 +171,25 @@ impl From<Key> for UserAction {
     }
   }
 }
+
+#[derive(Debug)]
+pub struct GameConfig {
+  pub food_count: usize,
+}
```
Comment (sashaweiss) [inline src/snek/driver.rs]: spacing

File: src/snek/driver.rs [hunk 3]
```diff
@@ -163,3 +171,25 @@ impl From<Key> for UserAction {
...
+  fn default() -> Self {
+    Self { food_count: 1 }
+  }
+}
+
+#[derive(Debug)]
+pub struct Config {
+  pub game_config: GameConfig,
+}
+impl Default for Config {
```
Comment (sashaweiss) [inline src/snek/driver.rs:191]: spacing

File: src/snek/driver.rs [hunk 4]
```diff
@@ -163,3 +171,25 @@ impl From<Key> for UserAction {
     }
   }
 }
+
+#[derive(Debug)]
```
Comment (sashaweiss) [inline src/snek/driver.rs]: can we move the struct definitions to up above the `impl Driver`? Would prefer to have all the type definitions in one place, and all the impls in another

**Review by sashaweiss — APPROVED** [review]

**Pull request merged**